### DISCUSSION TITLE
Support instantclick.io

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [livewire/livewire#3662] [3. Feature ideas]
Please support instant click (like turbolinks)
http://instantclick.io

**Comment by joshhanley** [comment]
@thanhtoan1196 Turbolinks support is just an adapter package. If you need it, I'd suggest writing your own adapter package for instant click.

If you're just looking for another alternative to Turbo, you could try [swupjs](https://swup.js.org/getting-started) who have already written their own Livewire adapter.
